providers/rackspace-cloudservers-us/src/test/java/org/jclouds/rackspace/cloudservers/us/compute/CloudServersUSTemplateBuilderLiveTest.java

```diff
@@ -39,11 +39,11 @@ public CloudServersUSTemplateBuilderLiveTest() {
 
    @Test
    public void testTemplateBuilder() {
-      Template defaultTemplate = this.view.getComputeService().templateBuilder().build();
+      Template defaultTemplate = this.view.getComputeService().templateBuilder().imageNameMatches(".*Ubuntu 14\\.10.*").build();
```

Comment:
Absolutely :) The important thing is to make sure there are no args to the template builder in this test. Thanks!